lib/fastlane/plugin/wpmreleasetoolkit/helper/github_helper.rb

```diff
@@ -187,6 +176,58 @@ def self.comment_on_pr(project_slug:, pr_number:, body:, reuse_identifier: Secur
 
         reuse_identifier
       end
+
+      # Update a milestone for a repository
+      #
+      # @param [String] repository The repository name (including the organization)
+      # @param [String] number The number of the milestone we want to fetch
+      # @param options [Hash] A customizable set of options.
+      # @option options [String] :title A unique title.
+      # @option options [String] :state
+      # @option options [String] :description A meaningful description
+      # @option options [Time] :due_on Set if the milestone has a due date
+      # @return [Milestone] A single milestone object
+      # @see http://developer.github.com/v3/issues/milestones/#update-a-milestone
+      #
+      def self.update_milestone(repository:, number:, options:)
+        client.update_milestone(repository, number, options)
+      end
+
+      # Unlock a single branch from a repository
```

Comment:
💄  "Unlock" might not be super accurate term here, because we can also decide to remove only part of the protection options but not all. So depending on the `options:` you pass and if there are still branch protections you didn't touch with that call and are still enabled on the branch, the branch might not be _completely_ unprotected after that call.
Besides, I feel it's clearer to use the same vocabulary as GitHub's UI (and API) here, with use terms around branch _protection_, not branch _locking_.

Reply:
Thanks for the tip! Indeed was not just a simple lock, have looked now into the docs.